timescaledb/overview/release-notes/index.md

```diff
@@ -20,6 +20,40 @@ Want to stay up-to-date with new releases? Subscribe to get
 *   1 step continuous aggregate policy management
 *   Migrate continuous aggregates to the new format
 
+## 2.8.1 (2022-10-06)
+
+This release is a patch release. We recommend that you upgrade at the
+next available opportunity.
+
+**Bugfixes**
```

Comment:
```suggestion
**Bug fixes**
```

Reply:
@iroussos Thanks for the hint. It is changed in the current version of the PR.